fancybox/asset/fancybox/fancybox.config.js

```diff
@@ -1,3 +1,4 @@
 $(document).ready(function() {
     $.fancybox.defaults.loop = "true";
+    $("body").off("click", ".wall-item-body a img");
```

Comment:
Please add a reference to the listener it deactivates, ideally the file and the line.

Reply:
Added comment for next PR